HW02_B_Mimic_starter.Rmd

```diff
@@ -69,7 +81,14 @@ set.seed(42)
 Now using the mpg dataset and the corvette dataset, make this graph:
 
 ```{r graoh 3 code}
-
+corvette$Model <- factor(corvette$model)
+levels(corvette$Model) <- "Corvette"

```

Comment:
The graph looks great! I am not sure if these two lines are needed? When I tried to re-run your code, I got the same graphs, with or without the two lines.

Reply:
I just used them to change "corvette" to "Corvette", but actually I could have done this by `paste("Corvette", year, sep", ")`. I made a detour.